arch/arm64/src/imx9/imx9_usbdev.c

```diff
@@ -54,6 +54,21 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
+#if !defined(ARMV8M_DCACHE_LINESIZE) || ARMV8M_DCACHE_LINESIZE == 0
```

Comment:
why check ARMV8M macro in arm64 arch?

Reply:
This is cortex-a8. Or do you mean why arm-v8m and not arm-v8a? This was just to be consistent with other drivers which use the same.

Reply:
Yes, it looks strange that 64bit cortex-a chip use ARMV8M_DCACHE_LINESIZE, but not ARMV8A_DCACHE_LINESIZE.

Reply:
Ok, I changed these. @pussuw, please update your current and upcoming PRs accordingly (currently pending https://github.com/apache/nuttx/pull/12183)

Reply:
Yes, that's an oops on my behalf